src/components/bookList/BookList.tsx

```diff
@@ -1,52 +1,69 @@
 import { useEffect, useState } from 'react'
 
+import { AxiosResponse } from 'axios'
 import InfiniteScroll from 'react-infinite-scroll-component'
 
 import { Book } from '../../model/Book'
-import BookRequest from '../../model/BookRequest'
 import BookService from '../../services/BookService'
 import Card from '../card/Card'
 import './BookList.css'
+import Search from '../search/Search'
+import Where from '../../model/Where'
+import BookResponse from '../../model/BookResponse'
 
-const numberOfCard = 24
+interface BooksRequest {
+  PageNumber: number,
+  PageLength: number,
+  Where: Where[],
+}
+const initialItem = 6
```

Comment:
Initial item what? Not sure what is this const used for

Reply:
This is initial number for render card to main page. I changed name for this const.